tools/slo-generator/slo_generator/utils.py

```diff
@@ -187,9 +190,7 @@ def import_dynamic(package, name, prefix="class"):
     except Exception as exception:  # pylint: disable=W0703
         LOGGER.error(
             f'{prefix.capitalize()} "{package}.{name}" not found, check '
-            f'package and class name are valid, or that importing it doesn\'t '
-            f'result in an exception.')
```

Comment:
Lint + running into this fails, can you please restore those two lines ?

Reply:
Sorry about that...